### ISSUE TITLE
TDS Waiver Mod: Internal Prototype

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#19568]
### Internal:
- Place the mod link in the 'Assessments' top menu

- Review area (auto-populated based on council district)
- Lead Reviewer 
    - Ph1 assign based on review area
    - Ph 2 Requesting assignment logic such as 'If it is council district 1 and waiver type is Other, it should be assigned to Kaylie'
- Waiver Type
- Fee info: Review Fee Sent, Review Fee Paid `No validation anywhere on this`
- Status: Application Incomplete, Needs Review, In Review, Waiting on Customer, Approved, Rejected

**Major Features:**
- User can reassign the lead reviewer
- Review Cycles: 
    - Ability to request a companion review from other reviewers (like Cap Metro)  `Discuss solution during APPS backlog grooming`
![image](https://github.com/user-attachments/assets/fd93ae9c-b34c-4437-9ba6-467d35cb0c49)
- Access code feature
- ID Generation Logic:` WVR-YY-123445`

**Comment by ChrispinP** [comment]
Staff Process has been built out. App-wide roles have been updated to Assessment Reviewer.

Accessed from the Assessments menu:
![Image](https://github.com/user-attachments/assets/1b281729-0df4-4419-945e-23e44c2c99a3)

TPW Waiver Assessments page:
![Image](https://github.com/user-attachments/assets/e22ee51a-d3af-466b-9269-ae830b173e62)

Similar Details menu, Meeting capabilities, Attachments, & Review cycles.
![Image](https://github.com/user-attachments/assets/18846b7c-b919-48f6-90e5-5181b3afb707)

Similar Review capabilities as well.

Ready for Testing

**Comment by ChrispinP** [comment]
Rebuild for staff views is complete and ready for testing again. Still need to work on the few additional enhancements to Statuses and multiple approvals for the cycles. There is some additional cleanup that needs to happen on the Cycle and Cycle content tables. The edit Case Details page may also need some adjustments.

**Comment by ChrispinP** [comment]
Staff now have separate menu's for the Lead & PM Reviewers on a cycle that have validation in addition to an anytime menu. Restrictions have been applied and rules updated on when cycles & companion reviews can be created, when reviews can be started/completed, and when notes can be modified. Case status should update appropriately once both reviews are completed and should notify the customer. Case approval will also now include access code information for the customer.

![Image](https://github.com/user-attachments/assets/63cbf4d3-69e2-4b66-ac47-da77b8bab30d)

ready for UAT

**Comment by TracyLinder** [comment]
This has been built out; any UAT feedback and iteration will be documented on in the UAT notes on a separate issue.
